watchguard_firebox_mr (#27037)
demisto/content: Packs/WatchguardFirebox/README.md

# Watchguard Firebox
This pack includes Cortex XSIAM content

# How to config to send syslog LEEF logs

> 1. Select System > Logging.

Reviewer: ```suggestion
1. Select **System** > **Logging**.
```

   The Logging page appears.
2. Select the Syslog Server tab.

> 3. Select the Send log messages to the syslog server at this IP address check box.

Reviewer: ```suggestion
3. Click the **Send log messages to the syslog server at this IP address** checkbox.
```

> 4. In the IP Address text box, type the IP address for the syslog.

Reviewer: ```suggestion
4. In the **IP Address** text box, type the IP address for the syslog.
```

5. In the Port text box, the default syslog server port (514) appears. To change the server port, type or select a different port for your server.

> 6. From the Log Format drop-down list, select IBM LEEF.

Reviewer: ```suggestion
6. From the **Log Format** drop-down list, select **IBM LEEF**.
```

![Server Screenshot](./doc_imgs/syslogleef.png)

8. In the Syslog Settings section, for each type of log message, select a syslog facility from the drop-down list.

>    If you select the IBM LEEF log format, you must select the The syslog header check box before you can select the syslog facility for the log message types.

Reviewer: ```suggestion
   If you select the IBM LEEF log format, you must select the syslog header checkbox before you can select the syslog facility for the log message types.
```

   For high-priority syslog messages, such as alarms, select Local0.

>    To assign priorities for other types of log messages (lower numbers have greater priority), select Local1–Local7.

Reviewer: ```suggestion
   To assign priorities for other types of log messages (lower numbers have greater priority), select **Local1–Local7**.
```
